.tool-versions

```diff
@@ -1 +1,4 @@
-golang 1.17.8
+golang 1.18
```

Comment:
All for this, but have we transitioned to 1.18 yet?

Reply:
We have for solana and terra so I assumed everywhere else had. I can pull us back to 1.17 if not.